pdf building dependency problem

When I build the pdf as documented here https://galaxyproject.github.io/training-material//topics/training/tutorials/create-new-tutorial-jekyll/tutorial.html I get the following error when I run `make pdf`:

`Dependency Error: Yikes! It looks like you don't have jemoji or one of its dependencies installed. In order to use Jekyll as currently configured, you'll need to install this gem. The full error message from Ruby is: 'libiconv.so.2: cannot open shared object file: No such file or directory - /home/berntm/anaconda_ete/envs/galaxy_training_material/lib/ruby/gems/2.4.0/gems/nokogiri-1.8.1/lib/nokogiri/nokogiri.so' If you run into trouble, you can find helpful resources at https://jekyllrb.com/help/!`

after installing `libiconv` with `conda install libiconv` it worked. 

During the build I get some errors:

```
Loading page http://localhost:4000/topics/sequence-analysis/tutorials/mapping/slides.html ...
+- Unable to load resource from URL: https://fonts.googleapis.com/css?family=Droid+Serif:400,700,400italic
|_ Error code: 6
|_ Description: SSL handshake failed
```

Now `make pdf` gives

```
Could not find gem 'awesome_bot' in rubygems repository https://rubygems.org/ or installed locally.
The source does not contain any versions of 'awesome_bot'
```

So I guessed that I need to run `make install` again. But it complained about missing npm which I installed with 

`conda install -c cpcloud npm`

Now `make install` gives the following errors:

```
npm install decktape
npm WARN engine decktape@2.6.1: wanted: {"node":">=7.10.0"} (current: {"node":"0.10.33","npm":"1.4.28"})
npm WARN engine chalk@2.3.0: wanted: {"node":">=4"} (current: {"node":"0.10.33","npm":"1.4.28"})
npm WARN engine puppeteer@0.13.0: wanted: {"node":">=6.4.0"} (current: {"node":"0.10.33","npm":"1.4.28"})
npm WARN engine ansi-styles@3.2.0: wanted: {"node":">=4"} (current: {"node":"0.10.33","npm":"1.4.28"})
npm WARN engine supports-color@4.5.0: wanted: {"node":">=4"} (current: {"node":"0.10.33","npm":"1.4.28"})
npm WARN engine mime@1.6.0: wanted: {"node":">=4"} (current: {"node":"0.10.33","npm":"1.4.28"})
npm WARN engine agent-base@4.1.2: wanted: {"node":">= 4.0.0"} (current: {"node":"0.10.33","npm":"1.4.28"})
|
> puppeteer@0.13.0 install /home/berntm/projects/training-material/node_modules/decktape/node_modules/puppeteer
> node install.js


/home/berntm/projects/training-material/node_modules/decktape/node_modules/puppeteer/install.js:60
  const cleanupOldVersions = allRevisions.map(({platform, revision}) => Downlo
                                                        ^
SyntaxError: Unexpected token ,
    at Module._compile (module.js:439:25)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:906:3
npm WARN package.json bcrypt-pbkdf@1.0.1 No repository field.
npm WARN package.json punycode@1.4.1 punycode is also the name of a node core module.

> hummus@1.0.83 install /home/berntm/projects/training-material/node_modules/decktape/node_modules/hummus
> node-pre-gyp install --fallback-to-build

[hummus] Success: "/home/berntm/projects/training-material/node_modules/decktape/node_modules/hummus/binding/hummus.node" is installed via remote
npm ERR! puppeteer@0.13.0 install: `node install.js`
npm ERR! Exit status 8
npm ERR! 
npm ERR! Failed at the puppeteer@0.13.0 install script.
npm ERR! This is most likely a problem with the puppeteer package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node install.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls puppeteer
npm ERR! There is likely additional logging output above.

npm ERR! System Linux 4.4.0-101-generic
npm ERR! command "/home/berntm/anaconda_ete/envs/galaxy_training_material/bin/node" "/home/berntm/anaconda_ete/envs/galaxy_training_material/bin/npm" "install" "decktape"
npm ERR! cwd /home/berntm/projects/training-material
npm ERR! node -v v0.10.33
npm ERR! npm -v 1.4.28
npm ERR! code ELIFECYCLE
npm ERR! not ok code 0
Makefile:56: recipe for target 'install' failed
make: *** [install] Error 1
```

Still got a dependency error: 

```
Dependency Error: Yikes! It looks like you don't have jemoji or one of its dependencies installed. In order to use Jekyll as currently configured, you'll need to install this gem. The full error message from Ruby is: 'libiconv.so.2: cannot open shared object file: No such file or directory - /home/berntm/anaconda_ete/envs/galaxy_training_material/lib/ruby/gems/2.4.0/gems/nokogiri-1.8.1/lib/nokogiri/nokogiri.so' If you run into trouble, you can find helpful resources at https://jekyllrb.com/help/! 
jekyll 3.6.2 | Error:  jemoji
Makefile:21: recipe for target 'detached-serve' failed
make: *** [detached-serve] Error 1
```

Solved it with `conda install -c r libiconv`

make pdf is running now. will report soon.

--- a/environment.yml
+++ b/environment.yml
@@ -14,3 +14,4 @@ dependencies:
   - zlib=1.2.8
   - yamllint=1.2.1
   - nodejs=8.8.1
+  - libiconv

